addons/pos_hr/tests/test_frontend.py

```diff
@@ -121,3 +121,23 @@ def test_basic_user_can_change_price(self):
             "test_basic_user_can_change_price",
             login="pos_user",
         )
+
+    def test_cashier_changed_in_receipt(self):
+        """
+        Checks that when the cashier is changed during the order,
+        the receipts displays the employee that concluded the order,
+        meaning the one that was at the register when the customer was paying.
+        Also checks that the order has the right cashier and employee in the same
+        use case.
+        """
+        self.product_a.available_in_pos = True
+        self.main_pos_config.with_user(self.pos_admin).open_ui()
+
+        self.start_tour(
+            "/pos/ui?config_id=%d" % self.main_pos_config.id,
+            "test_cashier_changed_in_receipt",
+            login="pos_admin",
+        )
+        order = self.env['pos.order'].search([('session_id', '=', self.main_pos_config.current_session_id.id)], limit=1)
```

Comment:
```suggestion
        order = self.main_pos_config.current_session_id.order_ids[0]
```
Maybe you should be careful with the order but here you only have one I think